Makefile

```diff
@@ -21,16 +21,18 @@ install-dev-tools:  ## Installs all necessary cargo helpers
 	cargo install cargo-udeps
 	cargo install flaky-finder
 	cargo install cargo-nextest --locked
+	cargo install cargo-risczero
+	cargo risczero install
 
 lint:  ## cargo check and clippy. Skip clippy on guest code since it's not supported by risc0
 	## fmt first, because it's the cheapest
-	cargo fmt --all --check
-	cargo check --all-targets --all-features
+	cargo +nightly fmt --all --check
+	cargo +nightly check --all-targets --all-features
```

Comment:
We only want `+nightly` for `cargo fmt`, all the other commands need to use stable.

Reply:
Done!